oak-it/src/test/java/org/apache/jackrabbit/oak/composite/CompositeNodeStoreTest.java

```diff
@@ -88,6 +86,9 @@
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
 
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
```

Comment:
shouldn't those 2 imports also be replaced by org.apache.jackrabbit.guava.common.* ?